source/transactions/tests/README.rst

```diff
@@ -640,3 +645,4 @@ Changelog
 :2019-02-13: Modify test format for 4.2 sharded transactions, including
              "useMultipleMongoses", ``object: testRunner``, the
              ``targetedFailPoint`` operation, and recoveryToken assertions.
+:2020-06-12: Add the default value for heartbeatFrequencyMS.
```

Comment:
I don't think any of these additions need a changelog entry, since we're just provide some advice and this not anything that requires driver changes.

Reply:
Agreed.